Source/WebCore/platform/graphics/gstreamer/InbandTextTrackPrivateGStreamer.cpp

```diff
@@ -131,7 +131,7 @@ void InbandTextTrackPrivateGStreamer::notifyTrackOfSample()
         ASSERT(isMainThread());
         ASSERT(!hasClients() || hasOneClient());
         notifyMainThreadClient([&](auto& client) {
-            downcast<InbandTextTrackPrivateClient>(client).parseWebVTTCueData(std::span { mappedBuffer.data(), mappedBuffer.size() });
+            downcast<InbandTextTrackPrivateClient>(client).parseWebVTTCueData(unsafeMakeSpan(mappedBuffer.data(), mappedBuffer.size()));
```

Comment:
Same as above, there is `GstBufferMapper::span()`:

```suggestion
            downcast<InbandTextTrackPrivateClient>(client).parseWebVTTCueData(mappedBuffer.span());
```